features/step_definitions/calabash_steps.rb

```diff
@@ -135,5 +135,13 @@
       wait_for_element_exists("* id:'#{id}' text:'#{text}'")
 end
 
+Then("I press {string} for {int} times") do |name,count| 
```

Comment:
if it bothers you to write "1 times", rather than "1 time", I believe you can write:

> ... {int} (?@time|times)")

Reply:
Thanks for the tip. I'm going to implement it -> {int} time(s)"